packages/luna/src/framework/engine/document-renderer.js

```diff
@@ -3,8 +3,7 @@ import { getSerializableConfig, getSettings, getManifest } from '../config.js';
 import posthtml from 'posthtml';
 import posthtmlInsertAt from 'posthtml-insert-at';
 
-import { InlineInject } from '../../decorators/service.js';
-import ComponentLoader from '../loaders/component-loader.js';
+	import ComponentLoader from '../loaders/component-loader.js';
```

Comment:
@lukas-schardt This is probably not a problem but there is some whitespace in front of the statement.